src/app/builder/builder.component.html

```diff
@@ -122,32 +132,78 @@
       <h4>Copy and Use Your Embed Code</h4>
 
       <p>Copy the code for one of the following players, and place it somewhere on your webpage.</p>
+      <section class="builder-step" id="step-4a" *ngIf="props">
+          <h4>
+            <input type="checkbox" [checked]="playPlaylist" (change)="togglePlayPlaylist()">
+            Or: Make It A Playlist
```

Comment:
I find this building interface to be a little bit confusing. Maybe that's how the designs lay this out though? Where are the designs for this? The only Embed designs I can find don't show the builder or a playlist https://drive.google.com/open?id=0B7nKGpH0IVlJS2ZReHQ2ODhYY0k

Just feedback on my impressions: It's not immediately clear to me that I need to select play the latest episode to get to the playlist feature. Could all of the options be presented instead of hanging the playlist option off the play latest option? Then the user would pick only one option instead of both for playlist.